src/core/public/styles/_index.scss

```diff
@@ -1,2 +1,3 @@
 @import './base';
 @import './ace_overrides';
+@import './chrome/index';
```

Comment:
The `scss` file related to the `chrome` domain that couldn't be loaded from their component (due to using core mixins or variables) were moved to `src/core/public/styles/chrome`